app/views/components/application_component.rb

```diff
@@ -9,10 +9,25 @@ class ApplicationComponent < Phlex::HTML
   include Phlex::Rails::Helpers::ImageTag
   include Phlex::Rails::Helpers::LinkTo
 
+  def initialize(**user_attrs)
```

Comment:
Here, I'm just being picky😅

For me the name `user_attrs` is functional, but it may not be the most descriptive or intuitive choice, especially in the context of a component. 

I'd stick with just `attrs`